libraries/AP_HAL_ChibiOS/hwdef/AEROFOX-H7/defaults.parm

```diff
@@ -0,0 +1,6 @@
+ARSPD_TYPE	8

```

Comment:
remove, unless you are supplying a Dronecan airspeed sensor with the autopilot to purchasers.